### ISSUE TITLE
Daily Mob Totems & Crates Glitch

### ISSUE THREAD

**Issue by SpiderBobPlayz** [Wynncraft/Issues#8481]
When I logged in yesterday I didn't get my daily crates & mob totems & the same today. plz help.

-Spider_Bob_Playz

**Comment by thecoolkogut4** [comment]
Have you previously gotten them?

**Comment by SpiderBobPlayz** [comment]
yes I have previously gotten them I took a few months break but I would alwayz get them by 10:00 am before the break but than when I got back it stopped working

**Comment by LordHelmchen98** [comment]
Are you still facing that problem?
